### PR TITLE
Enable session fingerprint and capabilities tests

### PR DESCRIPTION
**Pull request by manuel-alvarez-alvarez**
## Motivation

Enables the test for session fingerprints and also RC capabilities.

## Changes

Updates the java manifest and also adds the required scenario to enable the RC endpoint in the agent.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: tests/appsec/test_fingerprinting.py [hunk 1]
```diff
@@ -82,6 +83,7 @@ def test_session(self):
 
 @rfc("https://docs.google.com/document/d/1DivOa9XsCggmZVzMI57vyxH2_EBJ0-qqIkRHm_sEvSs/edit#heading=h.32nt1jz5tm2n")
 @features.fingerprinting
+@scenarios.remote_config_mocked_backend_asm_dd_nocache
```
Comment (manuel-alvarez-alvarez) [inline tests/appsec/test_fingerprinting.py:86]: Required to enable the RC endpoint in the agent.

**Comment by uurien** [comment]
You have changed the scenario, I think that it worth to check that the tests are running in the rest of the languages, I think that you should remove `[java]` from the title before merging the PR.

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
CI failing

**Comment by manuel-alvarez-alvarez** [comment]
> CI failing

Should be working fine now.

File: tests/appsec/test_fingerprinting.py (hunk 1 above)
Comment (cbeauchesne) [reply tests/appsec/test_fingerprinting.py:110]: with that, you'll run this test on the `remote_config_mocked_backend_asm_dd_nocache` scenario, but not on the default scenario, is it what you want to achieve ?

Comment (manuel-alvarez-alvarez) [reply tests/appsec/test_fingerprinting.py:110]: The default scenario does not come with the RC endpoint enabled in the agent, that's the reason for using this scenario. Is there a better way to do it?

**Review by cbeauchesne — APPROVED** [review]

**Review by robertomonteromiguel — APPROVED** [review]

File: tests/appsec/test_fingerprinting.py [hunk 2]
```diff
@@ -106,6 +107,7 @@ def test_session(self):
 
 @rfc("https://docs.google.com/document/d/1DivOa9XsCggmZVzMI57vyxH2_EBJ0-qqIkRHm_sEvSs/edit#heading=h.32nt1jz5tm2n")
 @features.fingerprinting
+@scenarios.remote_config_mocked_backend_asm_dd
```
Comment (NachoEchevarria) [inline tests/appsec/test_fingerprinting.py:110]: We also need this change for .Net :)

**Review by NachoEchevarria — APPROVED** [review]

File: tests/appsec/test_fingerprinting.py (hunk 2 above)
Comment (manuel-alvarez-alvarez) [reply tests/appsec/test_fingerprinting.py:110]: I've just rebased and will merge as soon as the CI passes :wink:

**Pull request merged**